cms/djangoapps/contentstore/management/commands/tests/test_backfill_orgs_and_org_courses.py

```diff
@@ -38,7 +38,8 @@ class BackfillOrgsAndOrgCoursesTest(SharedModuleStoreTestCase):
       `course_overviews`, `modulestore`, and `organizations`, respectively.
     """
 
-    def test_end_to_end(self):
+    @ddt.data("--dry", "--apply")
```

Comment:
The tests never caught this because they were only end-to-end testing the "real" run of the command. If we'd tested the dry run too, we would've caught the typo.